### ISSUE TITLE
More questions about pPromptFL

### ISSUE THREAD

**Issue by chenrxi** [PEILab-Federated-Learning/PromptFL#3]
I recently notice your new work pPromptFL in WWW23. 
I have some doubts about your experimental results. Assuming that each client is assigned non-overlapping category samples, for a well-pretrained large model, the personalization accuracy should not be so low. Using only local data to fine-tune the prompt words, the baseline algorithm should be close to 78%(for cross-silo setting, where 10 clients with non-overlapping classes, 5epoch 2shots, archtecture of image encoder is rn50), at least the results I reproduced are like this. So I'm not quite sure why this result occurs. As mentioned above, I hope the author can open source part of the code of pPromptFL to provide experimental results in line with the paper. Thank you so much.
Here are the results of your experiments：
![image](https://github.com/PEILab-Federated-Learning/PromptFL/assets/60959000/2e405377-f138-4bec-9e0d-5e4f6183bf20)

**Comment by 06Liz** [comment]
How dou you run these unexesting files?Such as sampling.py, draw.py, most of datasets.py.
